rpaas/acl.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2017 rpaas authors. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+import ipaddress
+import json
+import requests
+from networkapiclient import (Ip, Network)
+from requests.auth import HTTPBasicAuth
+
+
+class Dumb(object):
+
+    def __init__(self, storage):
+        self.storage = storage
+
+    def add_acl(self, name, src, dst):
+        src = str(ipaddress.ip_network(unicode(src)))
+        self.storage.store_acl_network(name, src, dst)
+
+    def remove_acl(self, name, src):
+        src = str(ipaddress.ip_network(unicode(src)))
+        self.storage.remove_acl_network(name, src)
+
+
+class AclApiError(Exception):
+    pass
+
+
+class AclManager(object):
+
+    def __init__(self, config, storage):
+        self.storage = storage
+        self.service_name = config.get("RPAAS_SERVICE_NAME", "rpaas")
+        self.acl_api_host = config.get("ACL_API_HOST")
+        self.acl_api_user = config.get("ACL_API_USER")
+        self.acl_api_password = config.get("ACL_API_PASSWORD")
+        self.acl_api_timeout = int(config.get("ACL_API_TIMEOUT", 30))
+        self.acl_port_range_start = config.get("ACL_PORT_RANGE_START", "32768")
```

Comment:
With docker swarm services this range starts in 30000, maybe change the default value?